src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $transition-duration transform $transition-type,
+      $transition-duration color $transition-type;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
+      &:hover {
+        transform: scale(1.2);
+        color: #34568b;
+      }
```

Comment:
your 4th card will not work, because you use in hover :not(:nth-child(4n))
check it and fix